packages/api-request-builder/src/query-search.js

```diff
@@ -35,7 +35,8 @@ export function fuzzy(): Object {
  * @return {Object} The instance of the service, can be chained.
  */
 export function fuzzyLevel(value: number): Object {
-  if (!value) throw new Error('Required argument for `fuzzyLevel` is missing')
+  if (!value && value !== 0)
```

Comment:
since we're evaluating two cases anyways, i would suggest to be more explicit by using 
```suggestion
  if (variable === undefined || variable === null)
```

Reply:
Sounds good to me 👍 Fixed here: https://github.com/commercetools/nodejs/pull/1371/commits/087f49d7636bac359e6c45d99584d6a72ce661d4